src/orquestra/sdk/_base/cli/_dorq/_ui/_presenters.py

```diff
@@ -302,20 +303,48 @@ def wf_list_for_prompt(self, wfs):
 
         return wfs, tabulated_labels
 
-    def workspaces_list_to_prompt(self, workspaces):
+    def workspaces_list_to_prompt(self, workspaces, config_name=None):
         # Create labels of workspaces that are printed by prompter
         # Label is <display_name> <id> tabulated nicely to create good-looking
         # table
         labels = [[ws.name, ws.workspace_id] for ws in workspaces]
+
+        if config_name == _config.AUTO_CONFIG_NAME:
+            try:
+                curr_ws = os.environ[_env.CURRENT_WORKSPACE_ENV]
+                for index, label in enumerate(labels):
+                    if label[1] == curr_ws:
+                        label.append("(CURRENT WORKSPACE)")
+                        # put current workspace at the top of the list so it is
+                        # auto-selected
+                        labels.insert(0, labels.pop(index))
+                        workspaces = workspaces[:]
+                        workspaces.insert(0, workspaces.pop(index))
+            except KeyError:
+                ...
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏I think `pass` makes more sense to say this is a no-op, whereas we have used `...` in other places to mean "this isn't implemented yet" (such as in protocol definitions).
```suggestion
                pass
```